backend/config/auth.config.test.js

```diff
@@ -1,4 +1,4 @@
 test('Environment variables are working as expected', () => {
   const backendUrl = process.env.REACT_APP_PROXY;
-  expect(backendUrl === `http://localhost:${process.env.BACKEND_PORT}`);
+  expect(backendUrl).toEqual(`http://localhost:${process.env.BACKEND_PORT}`);
```

Comment:
I was looking at toEqual and it is used for comparing objects, and not strings. This above is comparing two strings through string interpolation. I am pretty sure that is toBe() syntax instead. Both may work, and you can test this by trying to write a toEqual test that you know should fail and to see if it meets your expectations. Then rerun with a toBe() and compare. 


https://jestjs.io/docs/en/expect#toequalvalue
https://jestjs.io/docs/en/expect#tobevalue

Reply:
In this case you can use either toBe or toEqual. I prefer to use toEqual as I'm not checking for object ref. equality.

Reply:
@nickbeaird I changed to toBe